data.sql

```diff
@@ -34,3 +34,35 @@ UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
 UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
```

Comment:
- [ ] `required`
- [ ] Great job with the project so far! 👏 
- [ ] However, the requirements state that you must add screenshots of your queries in the `Pull Request` description.
- [ ] Please add the screenshots in the `Pull Request` description.